### DISCUSSION TITLE
Setup for react

### DISCUSSION THREAD

**Discussion by shvmdev** [tailwindlabs/tailwindcss#7350] [Help]
How to make setup of tailwind3.0 for react

**Comment by NovusOpus** [comment]
There's some Installation guides in the docs for react:

[Setting up Tailwind CSS with NextJS](https://tailwindcss.com/docs/guides/nextjs)

[Setting up Tailwind CSS with Create React App
](https://tailwindcss.com/docs/guides/create-react-app)

[Setting up Tailwind CSS with Gatsby](https://tailwindcss.com/docs/guides/gatsby)
